doc/client_lua_api.md

```diff
@@ -6,7 +6,7 @@ it's now called `core` due to the renaming of Luanti (formerly Minetest).
 `minetest` will keep existing as an alias, so that old code won't break.
 
 * More information at <http://www.minetest.net/>
-* Developer Wiki: <http://dev.minetest.net/>
+* Developer Wiki: <https://dev.luanti.org/>
```

Comment:
Is Developer Wiki an accurate characterization given the scope of content that was imported?

Reply:
it still calls itself "Luanti Developer Wiki" so :shrug:

Reply:
well, i suppose that can be dealt with later

Reply:
https://github.com/luanti-org/dev.luanti.org/issues/152